eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -620,6 +621,11 @@ class Channel(val nodeParams: NodeParams, wallet: EclairWallet, remoteNodeId: Pu
                   log.info(s"adding paymentHash=${u.paymentHash} cltvExpiry=${u.cltvExpiry} to htlcs db for commitNumber=$nextCommitNumber")
                   nodeParams.channelsDb.addOrUpdateHtlcInfo(d.channelId, nextCommitNumber, u.paymentHash, u.cltvExpiry)
               }
+              if (Helpers.aboveReserve(d.commitments) ^ Helpers.aboveReserve(commitments1)) {
```

Comment:
`Helpers.aboveReserve(d.commitments1) && !Helpers.aboveReserve(commitments)` is what we're actually testing for and would be easier to read

Reply:
I suppose you mean `!Helpers.aboveReserve(d.commitments) && Helpers.aboveReserve(commitments1)`